src/internal/operators/startWith.ts

```diff
@@ -41,37 +35,37 @@ export function startWith<T, D = T>(...array: Array<D | SchedulerLike>): Operato
  *
  * ## Examples
  *
- * Start the chain of emissions with `"first"`, `"second"`
+ * Emit a value when a timer starts.
  *
  * ```ts
- * import { of } from 'rxjs';
- * import { startWith } from 'rxjs/operators';
+ * import { timer } from 'rxjs';
+ * import { startWith, map } from 'rxjs/operators';
  *
- * of("from source")
- *   .pipe(startWith("first", "second"))
+ * timer(1000)
+ *   .pipe(
+ *     map(() => 'timer emit'),
+ *     startWith('timer start')
+ *   )
  *   .subscribe(x => console.log(x));
  *
  * // results:
- * //   "first"
- * //   "second"
- * //   "from source"
+ * // "timer start"
+ * // "timer emit"
  * ```
  *
- * @param {...T} values - Items you want the modified Observable to emit first.
- * @param {SchedulerLike} [scheduler] - A {@link SchedulerLike} to use for scheduling
- * the emissions of the `next` notifications.
- * @return {Observable} An Observable that emits the items in the specified Iterable and then emits the items
- * emitted by the source Observable.
- * @method startWith
- * @owner Observable
+ * @param values - Items you want the modified Observable to emit first.
+ *
+ * @see endWith
+ * @see finalize
+ * @see concat
  */
-export function startWith<T, D>(...array: Array<T | SchedulerLike>): OperatorFunction<T, T | D> {
-  const scheduler = array[array.length - 1] as SchedulerLike;
+export function startWith<T, D>(...values: Array<T | SchedulerLike>): OperatorFunction<T, T | D> {
```

Comment:
I see that https://github.com/ReactiveX/rxjs/pull/5157 has been merged, so I'm guessing the change in that PR will be picked up once this is rebased.